### PR TITLE
Kabocha on Rococo

### PR DESCRIPTION
**Pull request by decentration**

### CODE REVIEW

**Comment by decentration** [comment]
Adding logo and chain info for Kabocha on Rococo.

File: packages/apps-config/src/ui/colors.ts [hunk 1]
```diff
@@ -51,6 +51,7 @@ const chainHydrate = '#000000';
 const chainInterbtc = '#1a0a2d';
 const chainInterlay = '#3E96FF';
 const chainIpse = '#08a1e8';
+const chainKabocha = 'url(https://c.tenor.com/DPGJgNDVErgAAAAM/torus-vortex-spiral.gif) repeat'
```
Comment (jacogr) [inline packages/apps-config/src/ui/colors.ts:54]: Really don’t want external links here. More that can go wrong.

File: packages/apps-config/src/ui/colors.ts [hunk 2]
```diff
@@ -281,6 +282,10 @@ export const chainColors: Record<string, string> = Object.entries({
   IpseTestnet: chainIpse,
   'Jupiter A1': chainJupiter,
   'Jupiter PC1': chainJupiter,
+  Kabocha: chainKabocha,
+  'Kabocha (kabsoup) 🍲 🥫': chainKabocha,
```
Comment (jacogr) [inline packages/apps-config/src/ui/colors.ts:286]: Not convinced on the need for emojis /)

Comment (decentration) [reply packages/apps-config/src/ui/colors.ts:286]: Fair enough, will change these : D

**Comment by decentration** [comment]
@jacogr emojis have been removed and external url img changed : )

**Comment by jacogr** [comment]
Closing, conflicts, cannot be merged.

**Pull request closed without merging**